charts/rollups-validator-node/templates/localnode-job.yaml

```diff
@@ -18,10 +18,17 @@ spec:
       {{- with .Values.image.pullSecrets }}
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
+      serviceAccountName: {{ include "validator.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.validator.podSecurityContext | nindent 8 }}
       {{- end }}
       initContainers:
         - name: deployer
           image: {{ include "deployer.image" . }}
+          {{- if .Values.validator.localnode.deployer.command }}
+          command:
+          {{- include "tplvalues.render" (dict "value" .Values.validator.localnode.deployer.command "context" $) | nindent 12 }}
+          {{- end }}
```

Comment:
There's a problem here that I think may happen on other places.

In case the use replaces the image, the `args` and `command` may not work.

In case the user change the `command`, the args may not be suited and break the defined command.

We need a smart way to define a good behavior that doesn't breaks.

For ex.:

If I change the `dispatcher.command:` the `args:` used by the rollups-dispatcher binary should not be taken into consideration. But if the user wants to define other `args` or `extraArgs,` they'll be used.

Reply:
You could implement already this way here at  validator.localnode.deployer` and adjust the already defined services on another separate commit.

Reply:
I'm testing here without success.

I'm defining `.Values.validator.dispatcher.command: ["sleep", "infinity"]` but I don't get `command:` rendered in the `helm template` output.